packages/iov-bcp-types/src/atomicswapmerger.spec.ts

```diff
@@ -158,15 +158,15 @@ describe("AtomicSwapMerger", () => {
     const alice = "tiov1u8syu9juwx668k4vqfwl5vtm8j6yz89wamkcda" as Address;
     const bobAddress = "tiov1lpzdluzsq3u7tqkfkp3rmrfavkhv0ly56gjexe" as Address;
     const preimage = fromHex("00110011");
-    const hashLock = new Sha256(preimage).digest();
+    const hash = new Sha256(preimage).digest();
```

Comment:
Can we export a function to do this?

There is one in iov-bns/src/utils.ts:
```ts
export function preimageIdentifier(preimage: Uint8Array): HashId {
  const hash = new Sha256(preimage).digest();
  return hashIdentifier(hash);
}
```

maybe we can just move this into bcp(-types)?

Reply:
The whole idea of this identifier is BNS-specific. What we can do is expose a preimage->hash helper. And at that point the question arises if it is worth exposing a SHA56 hasher.

I think this starts to make sense as soon as miltiple algorithms are supported.

Reply:
Yeah, sorry, this Id is bns specific.
You are write about the hash being different.

But I would like to make something like 

```ts
export function hashPreimage(preimage: Uint8Array): Uint8Array {
  return new Sha256(preimage).digest();
}
``

Where we can add some client-side checks for the preimage length later, when we start enforcing them in the blockchain.

But sure, I guess we can leave it as is for now.